crates/provider-sdk/src/provider.rs

```diff
@@ -182,11 +184,10 @@ async fn subscribe_shutdown(
     quit: broadcast::Sender<()>,
     lattice: &str,
     provider_key: &str,
-    link_name: &str,
     host_id: &'static str,
 ) -> ProviderInitResult<mpsc::Receiver<oneshot::Sender<()>>> {
     let mut sub = nats
-        .subscribe(shutdown_subject(lattice, provider_key, link_name))
```

Comment:
note: as of 1.0 we've _always_ passed `"default"` as the link name for backwards compatibility. https://github.com/wasmCloud/wasmCloud/blob/main/crates/host/src/wasmbus/mod.rs#L2132

Reply:
it might be nice to put that as a comment somewhere in the code, if you think it'd confuse someone in this area